randovania/patching/prime/asset_conversion.py

```diff
@@ -252,17 +332,23 @@ def id_generator(asset_type):
         logging.info(f"Finished loading PAKs: {time.time() - start}")
 
         result = {}
-        for data in randomizer_data["ModelData"]:
-            if data["Model"] != Game.ECHOES.invalid_asset_id and data["AnimSet"] != Game.ECHOES.invalid_asset_id:
-                try:
-                    result[data["Name"]] = Asset(
-                        ancs=converter.convert_id(data["AnimSet"], Game.ECHOES, missing_assets_as_invalid=False),
-                        cmdl=converter.convert_id(data["Model"], Game.ECHOES, missing_assets_as_invalid=False),
-                        character=data["Character"],
-                        scale=data["Scale"][0],
-                    )
-                except (InvalidAssetId, UnknownAssetId) as e:
-                    logging.error("Unable to convert {}: {}".format(data["Name"], e))
+        assets_to_change = [
+            data
+            for data in randomizer_data["ModelData"]
+            if data["Model"] != Game.ECHOES.invalid_asset_id and data["AnimSet"] != Game.ECHOES.invalid_asset_id
+        ]
+
+        for i, data in enumerate(assets_to_change):
+            try:
+                status_update(f"Converting {data['Name']} from Prime 2", i / len(assets_to_change))
+                result["{}_{}".format(RandovaniaGame.METROID_PRIME_ECHOES.value, data["Name"])] = Asset(
+                    ancs=converter.convert_id(data["AnimSet"], Game.ECHOES, missing_assets_as_invalid=False),
+                    cmdl=converter.convert_id(data["Model"], Game.ECHOES, missing_assets_as_invalid=False),
+                    character=data["Character"],
+                    scale=data["Scale"][0],
+                )
+            except (InvalidAssetId, UnknownAssetId) as e:
+                logging.error("Unable to convert {}: {}".format(data["Name"], e))
```

Comment:
I think I'd go with fatal as well. This would mean a bad ISO.

Reply:
There's a few models that aren't in the vanilla game (translators, dark and light ammo expansions) that can cause this

Reply:
I'd prefer that at first we skip converting these, unless we figure out a way to consistently do it.